lib/iris/experimental/ugrid/mesh.py

```diff
@@ -2806,8 +2811,19 @@ def axis(self):
 
     @property
     def coord_system(self):
-        """The coordinate-system of a MeshCoord is always 'None'."""
-        return None
+        """The coordinate-system of a MeshCoord comes from its origin coords."""
+        select_kwargs = {
+            f"include_{self.location}s": True,
```

Comment:
On second thoughts, by analogy with #4860 this should probably *always* use "include_nodes".

Reply:
> by analogy with https://github.com/SciTools/iris/issues/4860 this should probably always use "include_nodes

No, that's the **_reverse_** of the change in #4860 !
This stays as it is.